yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -158,7 +159,7 @@ export interface PXE {
    * @throws If the code for the functions executed in this transaction has not been made available via `addContracts`.
    * Also throws if simulatePublic is true and public simulation reverts.
    */
-  proveTx(txRequest: TxExecutionRequest, simulatePublic: boolean, scopes?: AztecAddress[]): Promise<Tx>;
+  proveTx(txRequest: TxExecutionRequest, executionResult: PrivateExecutionResult): Promise<TxProvingResult>;
```

Comment:
Don't the descriptions of these functions need updating?

Reply:
Same as above